results.tex

```diff
@@ -2,35 +2,37 @@ \section{Results}
 
 \subsection{Nuclear Material Inventory}
 
-\Cref{tab:sim_result} lists  \gls{EU} inventory and
-history at year 2050, and the used \gls{UOX} will be
-reprocessed.
+\Cref{tab:sim_result1} lists \gls{EU} material inventory in 2050.
 
 \begin{table}[h]
 	\centering
 %	\scalebox{0.86}{
 		\begin{tabular}{cccc}
 			\hline
-			\textbf{Category} & \textbf{Unit} & \textbf{Value} & \textbf{Specifics}\\ 
-			\hline
-			Total UOX Usage & MTHM & 163,826 &  \\ 
-			Total MOX Usage & MTHM & 6,560 & \\ 
-			Total Used UOX Stored & MTHM & 125,453 & \gls{UNF} that is not reprocessed\\ 
-			Total Used  MOX Stored & MTHM & 3,438 & \gls{UNF} that is not reprocessed \\ 
-			Total Tailings & MTHM & 975,938 & \\ 
-			Total Natural U Used & MTHM & 1,146,420 & \\ \hline
+			\textbf{Category } & \textbf{Value} & \textbf{Unit} & \textbf{Specifics}\\ \hline
+			UOX Usage  & 158,794 & MTHM &  \\ 
+			MOX Usage  & 6,671 & MTHM & \\ 
+			\textbf{Used UOX Stored}  & \textbf{95,161} & MTHM & \gls{UNF} that is not reprocessed\\
+			\textbf{Used UOX Stored (France)} & \textbf{9,979} & MTHM & \gls{UNF} that is not reprocessed \\
+			Tails  & 979,463 & MTHM & \\ 
+			Natural U Used  & 1,141,916 & MTHM & \\ \hline
 		\end{tabular}
-		\caption{Listed are the metrics from the historical nuclear operation of \gls{EU} nations.
+		\caption{Nuclear material inventory in the \gls{EU} in 2050 is summarized. 
 				 The difference between total \gls{UOX} usage and \gls{UOX} stored is the amount
-				 that has been reprocessed for \gls{MOX}. Only the stored \gls{UOX} is used in the 
-				 second simulation.}
-		\label{tab:sim_result}
+				 that has been reprocessed for \gls{MOX}. Only the stored \gls{UOX} is used for \gls{ASTRID} fuel production.}
+		\label{tab:sim_result1}
 \end {table}
 \FloatBarrier
 
 
 Figures \ref{fig:eu_tail} and \ref{fig:eu_snf} show the 
-timeseries of tails and used fuel inventory accumulation in \gls{EU}.
+accumulation of tails and used fuel over time in \gls{EU}.
+Tails accumulate as a by-product of uranium enrichment. For every
+ton of \gls{UOX} fuel, about nine times of tails is produced. 
+Spent fuel is discharged from reactors every refueling period.
+The entire core is discharged when the reactor decommissions.
+A total of about $1,000,000 MTHM$ of tails and $100,000 MTHM$ of
+\gls{UNF} accumulate in 2050.
```

Comment:
This whole paragraph isn't necessary. Just keep the final sentence (though please edit it for clarity ... the UNF doesn't accumulate all at once "in" 2050 it accumulates "by". By "something substantive" I really just meant: tell the reader what they're supposed to see in the graph. In this case, "A total of about $1,000,000 MTHM$ of tails and $100,000 MTHM$ of \gls{UNF} have accumulated by 2050."